Composer/packages/client/src/pages/language-understanding/code-editor.tsx

```diff
@@ -39,9 +39,9 @@ const CodeEditor: React.FC<CodeEditorProps> = (props) => {
 
   const hash = props.location?.hash ?? '';
   const hashLine = querystring.parse(hash).L;
-  const line = Array.isArray(hashLine) ? +hashLine[0] : typeof hashLine === 'string' ? +hashLine : 0;
+  const line = Array.isArray(hashLine) ? parseInt(hashLine[0]) : typeof hashLine === 'string' ? parseInt(hashLine) : 0;
 
-  const inlineMode = !!intent;
+  const inlineMode = intent != null;
```

Comment:
Are you sure that this is the same intended behavior as before? `''` will now set `inlineMode` to true.